### PR TITLE
Title consistencies for NBA Jam Tournament Edition

### PR DESCRIPTION
**Pull request by mikehyrus**
Tournament Edition was the proper Subtitle used for the T unit releases of NBA Jam Tournament Edition while TE for short was used in some console releases.

### CODE REVIEW

**Comment by mikehyrus** [comment]
If you need me to add a colon or a dash or other changes needed let me know.

File: src/mame/midway/midtunit.cpp [hunk 1]
```diff
@@ -1695,11 +1695,11 @@ GAME( 1993, nbajam,     0,        tunit_adpcm, nbajam,   midtunit_state, init_nb
...
+GAME( 1994, nbajamte3,  nbajamte, tunit_adpcm, nbajamte, midtunit_state, init_nbajamte, ROT0, "Midway",   "NBA Jam Tournament Edition (rev 3.0 3/04/94)", MACHINE_SUPPORTS_SAVE )
+GAME( 1994, nbajamte3a, nbajamte, tunit_adpcm, nbajamte, midtunit_state, init_nbajamte, ROT0, "Midway",   "NBA Jam Tournament Edition (rev 3.0 2/26/94)", MACHINE_SUPPORTS_SAVE )
+GAME( 1994, nbajamte2,  nbajamte, tunit_adpcm, nbajamte, midtunit_state, init_nbajamte, ROT0, "Midway",   "NBA Jam Tournament Edition (rev 2.1 2/06/94)", MACHINE_SUPPORTS_SAVE )
+GAME( 1994, nbajamte2a, nbajamte, tunit_adpcm, nbajamte, midtunit_state, init_nbajamte, ROT0, "Midway",   "NBA Jam Tournament Edition (rev 2.0 1/28/94)", MACHINE_SUPPORTS_SAVE )
+GAME( 1994, nbajamte1,  nbajamte, tunit_adpcm, nbajamte, midtunit_state, init_nbajamte, ROT0, "Midway",   "NBA Jam Tournament Edition (rev 1.00 1/17/94)", MACHINE_SUPPORTS_SAVE )
 GAME( 1995, nbajamten,  nbajamte, tunit_adpcm, nbajamte, midtunit_state, init_nbajamte, ROT0, "Midway",   "NBA Jam T.E. Nani Edition (rev 5.2 8/11/95, prototype)", MACHINE_SUPPORTS_SAVE )
```
Comment (angelosa) [inline src/mame/midway/midtunit.cpp:1705]: - `NBA Jam: Tournament Edition`, which should also be fixed in hash folder for consistency (32x.xml, gameboy.xml, gamegear.xml, jaguar.xml, megadriv.xml, psx.xml, saturn.xml, snes.xml)
- Why nbajamten isn't fixed just below?

**Review by angelosa — CHANGES_REQUESTED** [review]

File: src/mame/midway/midtunit.cpp (hunk 1 above)
Comment (mikehyrus) [reply src/mame/midway/midtunit.cpp:1705]: Because I wasn’t sure what the Nani Edition was suppose to be renamed as yet.

Comment (angelosa) [reply src/mame/midway/midtunit.cpp]: Amazing, it's `T.E. Nani Edition` on POST, regular `NBA Jam: Tournament Edition` on title screen, `Nani NBA Jam T.E.` on "features" screen. 
Inclined to push the `Nani Edition` inside the parenthesis, `NBA Jam: Tournament Edition (Nani Edition, rev 5.2 8/11/95, prototype)` ...

Comment (mikehyrus) [reply src/mame/midway/midtunit.cpp]: Okay, I’ll take care of it right away.

**Comment by cuavas** [comment]
* The international Game Boy SNES, Mega Drive, Jaguar, Saturn and PlayStation versions say “NBA JAM T.E.” on the box art with “TOURNAMENT EDITION” in much smaller text underneath.
* The Japanese Mega Drive version has the “NBA JAM T.E.” and has “NBA ジム トーナメントエディション” underneath in relatively small text.  It doesn’t say “tournament edition” in Latin script anywhere.
* The Japanese PlayStation version does have “TOURNAMENT EDITION” in very small text.
* If you look at the legal notice screen on e.g. the Jaguar version,  it just says “NBA JAM™ TOURNAMENT EDITION” (no separation, no indication that “tournament edition” is a subtitle, it’s just part of the main title).  Similarly, the Mega Drive version’s legal notice screen says “NBA JAM TOURNAMENT EDITION™” – the trademark symbol is in a different place, but once again there’s no indication that “tournament edition” is not part of the title itself.

The “T.E.” abbreviation wasn’t used on the arcade cabinet side art or marquee, but it was used heavily in the marketing material and box art for *all* the console releases.

**Comment by mikehyrus** [comment]
I’ll take care of that later between daytime and evening Vas. 😴

**Comment by mikehyrus** [comment]
@cuavas Should I leave this as is or should I revert the - (dash) back on.

**Pull request merged**